addons/web/static/src/views/list/list_controller.js

```diff
@@ -365,7 +365,7 @@ export class ListController extends Component {
                 callback: () => this.duplicateRecords(),
             },
             delete: {
-                isAvailable: () => this.activeActions.delete && !isM2MGrouped,
+                isAvailable: () => this.activeActions.delete,
```

Comment:
why do we keep it for the duplicate action?

Reply:
You're massively out of date here. The duplicate feature has been added in Septembre. You should rebase the branch

Reply:
The duplicate feature is also there in the code, i just haven't made any changes in it as archive, unarchive and delete were specifically mentioned in the specs of the task. I think in the task it was meant to enable all 4 of the functions and I might have misunderstood it for the 3 because of the specs. So should we enable the duplicate too for the m2m grouping?